<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actionExpandAll"/>
    <addaction name="actionCollapseAll"/>
    <addaction name="separator"/>
    <addaction name="actionCompareView"/>
    <addaction name="actionDirView"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>选择列</string>
    </property>
    <addaction name="actionColName"/>
    <addaction name="actionColRefCount"/>
    <addaction name="actionColValue"/>
    <addaction name="actionColType"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>复制数据</string>
    </property>
    <addaction name="actionCopyName"/>
    <addaction name="actionCopyValue"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>配置</string>
    </property>
    <addaction name="actionRootDir"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="actionTreeView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>引用树</string>
   </property>
  </action>
  <action name="actionRootDir">
   <property name="text">
    <string>设置路径</string>
   </property>
  </action>
  <action name="actionCompareView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>对比视图</string>
   </property>
   <property name="toolTip">
    <string>对比视图</string>
   </property>
  </action>
  <action name="actionDirView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>目录树</string>
   </property>
  </action>
  <action name="actionExpandAll">
   <property name="text">
    <string>展开所有</string>
   </property>
   <property name="toolTip">
    <string>展开所有</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+=</string>
   </property>
  </action>
  <action name="actionCollapseAll">
   <property name="text">
    <string>收起所有</string>
   </property>
   <property name="toolTip">
    <string>收起所有</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+-</string>
   </property>
  </action>
  <action name="actionColName">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </action>
  <action name="actionColRefCount">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>引用计数</string>
   </property>
  </action>
  <action name="actionColValue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>值</string>
   </property>
  </action>
  <action name="actionColType">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>类型</string>
   </property>
  </action>
  <action name="actionCopyName">
   <property name="text">
    <string>名称</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCopyValue">
   <property name="text">
    <string>值</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
